nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -207,7 +207,7 @@
 
 - `grafana` has been updated to version 11.1. This version doesn't support setting `http_addr` to a hostname anymore, an IP address is expected.
 
-- `deno` has been updated to v2 which has breaking changes. Upstream will be abandoning v1 soon but for now you can use `deno_1` if you are yet to migrate (will be removed prior to cutting a final 24.11 release).
+- `deno` has been updated to v2 which has breaking changes.
```

Comment:
Maybe link the migration guide? e.g.

```md
- `deno` has been updated to v2 which has [breaking changes](https://docs.deno.com/runtime/reference/migration_guide/).
```

Reply:
Done.

Reply:
Can I mention that they made the dinosaur less cute too

Reply:
Yes.